packages/cfd/src/features/Components/proof-of-address.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FormikValues } from 'formik/dist/types';
+import ProofOfAddressForm from '@deriv/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form';
+
+type TProofOfAddress = {
+    index: number;
+    onSave: (index: number, values: FormikValues) => void;
+    onSubmit: (index: number, values: FormikValues) => void;
+};
+
+const ProofOfAddress = ({ index, onSave, onSubmit }: TProofOfAddress) => {
```

Comment:
Isn't it confusing to have the same name in different packages?

Reply:
We previously had it as cfd-poa but since we are inside the cfd package, it does not make sense to keep the `cfd` prefix. we can keep the `cfd` prefix for the `POA` and `POI` to avoid confusion. What do you suggest ?

Reply:
Should be okay, we can keep it for now.
By right, we should not create a connection between sibling packages and we will need to address this in the next phase of our refactoring.